modules/claims_api/app/models/claims_api/evidence_waiver_submission.rb

```diff
@@ -8,4 +8,12 @@ class ClaimsApi::EvidenceWaiverSubmission < ApplicationRecord
 
   has_kms_key
   has_encrypted :auth_headers, key: :kms_key, **lockbox_options
+
+  PENDING = 'pending'
+  SUBMITTED = 'submitted'
+  UPLOADED = 'uploaded'
+  UPDATED = 'updated'
```

Comment:
what's our differences here? since we're going to be updating in BGS and sending a pdf to VBMS, curious what the flow will be between these two. Tough to distinguish what order they'll exist in.

Reply:
`upload` for when the form has been uploaded ?
`updated` for when `filed5103_waiver_ind` is set to True